src/components/staking/PageValidator.vue

```diff
@@ -114,25 +114,25 @@
         </li>
         <li>
           <h4>Uptime</h4>
-          <span id="page-profile__uptime">
-            {{ validator.uptimePercentage | percent }}
-          </span>
+          <span id="page-profile__uptime">{{
+            chooseFilter(validator.uptimePercentage)
+          }}</span>
         </li>
         <li>
           <h4>Current Commission Rate</h4>
-          <span>{{ validator.commission | percent }}</span>
+          <span>{{ chooseFilter(validator.commission) }}</span>
         </li>
         <li>
           <h4>Max Commission Rate</h4>
-          <span>{{ validator.maxCommission | percent }}</span>
+          <span>{{ chooseFilter(validator.maxCommission) }}</span>
         </li>
         <li>
           <h4>Max Daily Commission Change</h4>
-          <span>{{ validator.maxChangeCommission | percent }}</span>
+          <span>{{ chooseFilter(validator.maxChangeCommission) }}</span>
         </li>
         <li>
           <h4>Last Commission Change</h4>
-          <span>{{ validator.commissionUpdateTime | fromNow }}</span>
+          <span>{{ chooseFilter(validator.commissionLastUpdate) }}</span>
```

Comment:
this would use the percent filter. remember the thing I said about setting the filter as the second parameter. maybe investigate if filters can act on the initial value

Reply:
Thank you. I just changed it to accept a second parameter. Using either of the filters first doesn't work. They are incompatible.